transceiver-collector/collector.go

```diff
@@ -317,6 +322,12 @@ func (t *TransceiverCollector) getMonitoredInterfaces() ([]string, error) {
 		if InterfacesIncluded && !contains(t.includeInterfaces, iface.Name) {
 			continue
 		}
+		if len(t.excludeInterfacesRegex.String()) > 0 && t.excludeInterfacesRegex.MatchString(iface.Name) {
```

Comment:
This can be `t.excludeInterfacesRegex != nil` instead of this, after reworking `compileRegexFlags`.